rpa-home/modules/ROOT/pages/automation-security-securesessions.adoc

```diff
@@ -0,0 +1,53 @@
+= Secure Sessions
+
+In RPA automations, a bot simulates a human user. The bot logs in to a system, sees the screen, and can interact with the UI via keystrokes and mouse actions. The bot can even interact with legacy systems that do not provide an API.
+
+The bot logs in to the computer, where it is installed. But, of course, it does not sit on a chair in front of the desktop. Nevertheless, if you enter the room where the computer is, you can see what the bot is doing: The screen shows the open applications, the mouse moves around and typing appears in forms. In MuleSoft RPA, we call this a _Desktop Session_. To prevent unauthorized data access or manipulation, you must secure the physical room.
+
+Or you can use what MuleSoft RPA calls _Secure Sessions_.
+
+Secure Sessions are hidden Desktop sessions that run in the background and are accessible only to the bot. An additional advantage of using Secure Sessions is that the bot can multiply itself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
+
+== General Considerations
+
+* A bot either can run in Desktop Session or Secure Session mode. You can change the mode while no configurations are deployed to the bot.
+* You can't switch the session mode while configurations are deployed on the bot.
+* Run configurations specify a session mode, too.
+* You can only deploy configurations to a bot that is in the same session mode.
+* You can specify the login data for the bot and the maximum number of Secure Sessions that the bot can run in parallel in the RPA Bot Configurator or in the RPA Manager.
+** The maximum number of Secure Sessions specified in the RPA Bot Configurator limits overwrites the same setting in RPA Manager.
++
+Enabling the same setting via the RPA Bot Configurator, limits but not not overwrites this setting.
```

Comment:
```suggestion
Enabling the same setting via the RPA Bot Configurator limits but does not overwrite this setting.
```

Reply:
Replaced with Hanna's suggestion from another file (organizing these informations and implementimng them as partials will be part of the Doc Space Redesign project.)